docs/pipeline-components-and-applications/loaders-storage-targets/snowplow-rdb-loader/upgrade-guides/6-0-0-upgrade-guide/index.md

```diff
@@ -131,14 +136,15 @@ igluctl table-check \
         --dbname  <database> \
         --dbschema <schema>
 # ...
-# * Comment problem - SchemaKey found in table comment [iglu:com.test/test/jsonschema/1-0-0] does not match expected [iglu:com.test/test/jsonschema/1-0-1]
-# * Column doesn't match, expected: 'wrong_type BIGINT', actual: 'wrong_type VARCHAR(4096)'
-# * Column existing in the storage but is not defined in the schema: 'only_in_storage VARCHAR(4096)'
+# * Column doesn't match, expected: 'example_field SMALLINT', actual: 'example_field INT'
 # ...   
 ```
 
+We got `Column doesn't match` output with the above example because, the table column had been migrated manually from `SMALLINT` to `INT`. Since the latest schema version that doesn't contain any breaking change is `1-0-0`, `table-check` command expects to see `SMALLINT` in the table therefore it gives `Column doesn't match` output.
```

Comment:
```suggestion
We got the `Column doesn't match` output with the above example because the table column had been migrated manually from `SMALLINT` to `INT`. Since the latest schema version that doesn't contain any breaking change is `1-0-0`, `table-check` command expects to see `SMALLINT` in the table therefore it gives the `Column doesn't match` output.
```